app/lib/Plugins/SearchEngine/Elastic8/FieldTypes/Length.php

```diff
@@ -39,15 +39,10 @@
 
 class Length extends GenericElement {
 
-	public function __construct($table_name, $element_code) {
-		parent::__construct($table_name, $element_code);
-	}
-
 	public function getIndexingFragment($content, array $options): array {
-		if (is_array($content)) {
-			$content = serialize($content);
-		}
-		if ($content == '') {
+		$this->setDefaultSuffix(self::SUFFIX_DOUBLE);
```

Comment:
Would it make more sense to set this as a class property or in a construct? to ensure the default value is set consistently.